mezzanine/bin/runtests.py

```diff
@@ -63,7 +64,9 @@ def cleanup_test_settings():
     django.setup()
 
     from django.core.management.commands import test
-    sys.exit(test.Command().execute(verbosity=1))
+    if django.VERSION < (1, 10):
+        sys.exit(test.Command().execute(verbosity=1))
+    sys.exit(call_command(test.Command(), verbosity=1))
 
```

Comment:
Defaults weren't being set properly in 1.10 when calling the command directly so I was getting an error running the tests. Might be connected to dropping optparse support, but I didn't dig too deep really.